NetPyNE-UI

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 CONFIG=$(pwd)/.jupyter-config
 export JUPYTER_CONFIG_DIR=$CONFIG
-exec jupyter notebook --NotebookApp.default_url=/geppetto --NotebookApp.token='' --library=netpyne_ui --NotebookApp.disable_check_xsrf=True
+exec jupyter notebook --NotebookApp.default_url=/geppetto --NotebookApp.token='' --NotebookApp.notebook_dir=workspace --library=netpyne_ui --NotebookApp.disable_check_xsrf=True
```

Comment:
@aranega this is the key change that makes netpyne ui chdir to the osb workspace